README.md

```diff
@@ -86,8 +86,9 @@ The *dist/* directory contains various CI/CD related files.
 #### Openshift Dev
 * Uses *dist/Dockerfile.build/Dockerfile* as the build container image
 * Run the following scripts on PR
-    * `dist/cargo_test.sh`
     * `dist/prow_yaml_lint.sh`
+    * `dist/prow_rustfmt.sh`
+    * `dist/cargo_test.sh`
```

Comment:
Out of curiosity, why the reorder?

Reply:
Probably its not needed, but I'd prefer to run static checks before unit tests, so that obvious errors would be caught first